cli/cmd/cluster.go

```diff
@@ -244,14 +309,30 @@ var _downCmd = &cobra.Command{
 		}
 		warnIfNotAdmin(awsClient)
 
-		prompt.YesOrExit(fmt.Sprintf("your cluster (%s in %s) will be spun down and all apis will be deleted, are you sure you want to continue?", *accessConfig.ClusterName, *accessConfig.Region), "", "")
+		clusterState, err := clusterstate.GetClusterState(awsClient, accessConfig)
+		if err != nil {
+			if errors.GetKind(err) == clusterstate.ErrUnexpectedCloudFormationStatus {
+				fmt.Println(clusterState.TableString())
+				fmt.Println(fmt.Sprintf("cluster %s in %s is in an unexpected state, please delete the cloudformation stacks manually on your AWS console %s", *accessConfig.ClusterName, *accessConfig.Region, getCloudFormationURLWithAccessConfig(accessConfig)))
+			}
+			exit.Error(err)
+		}
+
+		switch clusterState.Status {
+		case clusterstate.StatusNotFound:
+			exit.Error(errors.Wrap(ErrorClusterDoesNotExist(*accessConfig.ClusterName, *accessConfig.Region), "cluster down"))
```

Comment:
I'm not sure if we need the "cluster down" wrapping (and similar wrapping in the other commands), since the user knows what command they just typed, we can discuss